pkg/errors/errors.go

```diff
@@ -210,6 +323,13 @@ func GetGRPCError(err error) error {
 	return err
 }
 
+// IsMocErrorCode wraps a call to GetMocErrorCode. It returns true only
+// if the error has the same MocCode as the given code (no string matching).
+func IsMocErrorCode(err error, code moccodes.MocCode) bool {
```

Comment:
Could you please point me to an example usage of this function?

Reply:
https://github.com/microsoft/wssdcloudagent/blob/dccc5654228f3bd3f0ea4b647ba9998075c1152c/pkg/cloudagent/services/storage/virtualharddisk/virtualharddisk.go#L179

Reply:
For ref the example usage is
` errors.IsMocErrorCode(err, codes.NotFound) `

What's the difference between this and IsNotFound?

Reply:
IsNotFound is the old function, which will employ string matching. Whereas IsMocErrorCode only matches against the error code (won't match otherwise) in the root error.